<ui version="4.0" >
 <author></author>
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>workOrderMaterials</class>
 <widget class="QWidget" name="workOrderMaterials" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Maintain Work Order Materials</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>12</number>
     </property>
     <property name="topMargin" >
      <number>12</number>
     </property>
     <property name="rightMargin" >
      <number>12</number>
     </property>
     <property name="bottomMargin" >
      <number>12</number>
     </property>
     <property name="horizontalSpacing" >
      <number>12</number>
     </property>
     <property name="verticalSpacing" >
      <number>12</number>
     </property>
     <item row="0" column="0" >
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="WoCluster" name="_wo" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>290</width>
           <height>130</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="_close" >
           <property name="text" >
            <string>&amp;Close</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>100</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0" >
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="XLabel" name="_materialRequirementsLit" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string>Material Requirements:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <layout class="QVBoxLayout" >
           <item>
            <widget class="XTreeWidget" name="_womatl" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>0</width>
               <height>100</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" >
             <item row="0" column="1" >
              <widget class="XLabel" name="_numberOfItemsLit" >
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text" >
                <string>Number Of Items</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="2" >
              <widget class="XLabel" name="_totalQtyPerLit" >
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text" >
                <string>Total Qty. Per</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0" >
              <widget class="XLabel" name="_PickListItemsLit" >
               <property name="text" >
                <string>Pick List Items:</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1" >
              <widget class="XLabel" name="_pickNumber" >
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text" >
                <string/>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="2" >
              <widget class="XLabel" name="_pickQtyPer" >
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text" >
                <string/>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="0" >
              <widget class="XLabel" name="_nonPickListItems" >
               <property name="text" >
                <string>Non-Pick List Items:</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="1" >
              <widget class="XLabel" name="_nonPickNumber" >
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text" >
                <string/>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="2" >
              <widget class="XLabel" name="_nonPickQtyPer" >
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text" >
                <string/>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="0" >
              <widget class="XLabel" name="_totalLit" >
               <property name="text" >
                <string>Totals:</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="1" >
              <widget class="XLabel" name="_totalNumber" >
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text" >
                <string/>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="2" >
              <widget class="XLabel" name="_totalQtyPer" >
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text" >
                <string/>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="4" >
              <widget class="XLabel" name="_currentStdCostLit" >
               <property name="text" >
                <string>Current Std. Material Cost:</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="4" >
              <widget class="XLabel" name="_currentActCostLit" >
               <property name="text" >
                <string>Current Act. Material Cost:</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="4" >
              <widget class="XLabel" name="_maxCostLit" >
               <property name="text" >
                <string>Maximum Desired Cost:</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="5" >
              <widget class="XLabel" name="_currentStdCost" >
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text" >
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="5" >
              <widget class="XLabel" name="_currentActCost" >
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text" >
                <string/>
               </property>
              </widget>
             </item>
             <item row="3" column="5" >
              <widget class="XLabel" name="_maxCost" >
               <property name="minimumSize" >
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text" >
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="3" >
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" >
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" >
             <property name="spacing" >
              <number>5</number>
             </property>
             <item>
              <layout class="QVBoxLayout" >
               <property name="spacing" >
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="_new" >
                 <property name="text" >
                  <string>&amp;New</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="_edit" >
                 <property name="enabled" >
                  <bool>false</bool>
                 </property>
                 <property name="text" >
                  <string>&amp;Edit</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="_view" >
                 <property name="enabled" >
                  <bool>false</bool>
                 </property>
                 <property name="text" >
                  <string>&amp;View</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="_delete" >
                 <property name="enabled" >
                  <bool>false</bool>
                 </property>
                 <property name="text" >
                  <string>&amp;Delete</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="5" margin="5" />
 <customwidgets>
  <customwidget>
   <class>WoCluster</class>
   <extends>QWidget</extends>
   <header>wocluster.h</header>
  </customwidget>
  <customwidget>
   <class>XLabel</class>
   <extends>QLabel</extends>
   <header>xlabel.h</header>
  </customwidget>
  <customwidget>
   <class>XTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>xtreewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_wo</tabstop>
  <tabstop>_womatl</tabstop>
  <tabstop>_new</tabstop>
  <tabstop>_edit</tabstop>
  <tabstop>_view</tabstop>
  <tabstop>_delete</tabstop>
  <tabstop>_close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_close</sender>
   <signal>clicked()</signal>
   <receiver>workOrderMaterials</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_womatl</sender>
   <signal>valid(bool)</signal>
   <receiver>_view</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
